app/src/main/java/com/example/keybindhelper/ui/keybind/KeybindFragment.kt

```diff
@@ -67,19 +67,19 @@ class KeybindFragment : Fragment() {
 
             true;
         }
-        mainActivity.Menu!!.findItem(R.id.action_sub_hide_keybinds).setOnMenuItemClickListener {
+        mainActivity.menu!!.findItem(R.id.action_sub_hide_keybinds).setOnMenuItemClickListener {
             for(v in rv.children)
                 v.findViewById<RecyclerView>(R.id.keybind_zone).isVisible = false;
             true;
         }
-        mainActivity.Menu!!.findItem(R.id.action_sub_show_keybinds).setOnMenuItemClickListener {
+        mainActivity.menu!!.findItem(R.id.action_sub_show_keybinds).setOnMenuItemClickListener {
             for(v in rv.children)
                 v.findViewById<RecyclerView>(R.id.keybind_zone).isVisible=true;
             true;
         }
-        mainActivity.Menu!!.findItem(R.id.action_add).setOnMenuItemClickListener {
+        mainActivity.menu!!.findItem(R.id.action_add).setOnMenuItemClickListener {
```

Comment:
While you're here, avoid force unwrap (!!) at all costs.  That can create a NullPointerException, if the variable value is null.
Use something null safe, like foo?.let{} ?: instead.